docs/ceph.md

```diff
@@ -0,0 +1,90 @@
+Commands in this doc are an alternate to running the toolbox pod.
+
+# Ceph Commnads
```

Comment:
I was thinking to have the level 1 header at the top, then each level-2 header would have the name of the command, like this:
```
# Ceph Commnads

Commands in this doc are an alternate to running the toolbox pod.

## ceph
```

Reply:
DOne